cvat/apps/annotation/ddln_spotter.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (C) 2020
+
+format_spec = {
+    "name": "DDLN_CSV_BB",
+    "dumpers": [
+        {
+            "display_name": "{name} {format} {version} for images [BB]",
+            "format": "ZIP",
+            "version": "0.9",
+            "handler": "dump"
+        }
+    ],
+    "loaders": [
+        {
+            "display_name": "{name} {format} {version} for images [BB]",
+            "format": "ZIP",
+            "version": "0.9",
+            "handler": "load",
+        }
+    ],
+}
+
+def extractFileParams(filename):
+    directoryName, csvFilename = os.path.split(filename)
+    if directoryName:
+        directoryName = directoryName.split("/")[2]
+    # for local testing
+    else: 
+        directoryName = "dummy"
+    csvFilename = os.path.splitext(csvFilename)[0] + "_y_bb.csv"
+	
+    return directoryName,csvFilename
+
+def writeToCsv(dirname, filename, data):
+    try:
+        if not os.path.exists(dirname):
+            os.mkdir(dirname)
+        with open(os.path.join(dirname, filename), 'w', newline='') as csvfile:
+            csvfile.write(data)            
+    except Exception as e:
+        print("Error saving {}: {}".format(filename, e))
```

Comment:
nit: I think CVAT uses Python 3+, so you can use shorter format notation: https://docs.python.org/3/library/string.html#format-string-syntax

Reply:
Unfortunately Python f-strings doesn't work with 3.5.2 version, which is used by CVAT. Though we can consider upgrading to 3.6+ in some future.